finalPrep/modZeroHero.js

```diff
@@ -1,41 +1,61 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
+var heroName;
+var specialAbility;
 
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
+var greeting = 'Have no fear, ' + heroName + ' is here!';
+var catchphrase = `I just so happen to have the ability to ${specialAbility}`;
 
 // Declare two variables - power AND energy - set to integers
+var power = 9000;
+var energy = 173;
 
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
-
+var fullPower = power * 500;
+var fullEnergy = energy + 150
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
-
+var isHuman = false;
+var identityConcealed = true;
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
-
+var archEnemies = ['Bad Horse', 'Tacos R. Gonezo', 'Silent Butt-Deadly'];
+var sidekicks = ['Left Man', 'Hero Girl', 'Amazing Cat Face'];
 // Print the first sidekick to your console
-
+console.log(sidekicks[0]);
 // Print the last archEnemy to the console
-
+console.log(archEnemies[2]);
 // Write some code to add a new archEnemy to the archEnemies array
-
+archEnemies.push('Shadow Stalker');
 // Print the archEnemies array to console to ensure you added a new archEnemy
-
+console.log(archEnemies);
 // Remove the first sidekick from the sidekicks array
-
-// Print the sidekicks array to console to ensure you added a new sidekick
-
+sidekicks.shift();
+// Print the sidekicks array to console to ensure you removed a sidekick
+console.log(sidekicks);
 // Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
 //   - dangerLevel should be an integer
-//   - saveTheDay should be a string a hero would say once they save the day 
+//   - saveTheDay should be a string a hero would say once they save the day
 //   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
-
+function assessSituation(dangerLevel,saveTheDay,badExcuse) {
```

Comment:
Great work. Your use of indentation follows JS conventions and makes this readable. Also - strong solution for the conditional as it does not include any redundant code!

Very small - most style guides will call for a line of whitespace between lines 50 and 51, and definitely a space between `if` and `(` on line 51 🙂

Reply:
Added white space to line 51.